src/MusicStore/Platform.cs

```diff
@@ -82,5 +82,14 @@ public bool IsRunningOnNanoServer
                 return _isNano.Value;
             }
         }
+
+        //Sql client not available on mono, non-windows, or nano
+        public bool UseInMemoryStore
```

Comment:
This is the wrong place for this property. Platform has nothing to do with stores